packages/babel-plugin-transform-optional-catch-binding/src/index.js

```diff
@@ -0,0 +1,13 @@
+// import syntaxOptionalCatchBinding from "babel-plugin-syntax-optional-catch-binding";
+
+export default function() {
+  return {
+    visitor: {
+      CatchClause({ node }) {
+        if (!node.param) {
+          node.param.name = "_unused";
```

Comment:
This might conflict with something; also, if `node.param` is null or undefined, trying to set `node.param.name` will throw an error. You probably want [`generateUidIdentifier`](https://github.com/thejameskyle/babel-handbook/blob/master/translations/en/plugin-handbook.md#toc-generating-a-uid.)